BL-5229 Make Android method choices more obvious

JohnThomson opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/bloombooks/bloomdesktop/1957)
<!-- Reviewable:end -->

hatton commented:
This is great.
UI-wise, I see these issues
1) the "method-option-wrapper " should be background-color: white
2) that pesky horizontal line you mentioned to me
3) In the dropdown mode, images don't show in either Chrome or Nightly... so this will break someday when we upgrade.

Name wise, while we're in here...  "method-choices" would parallel the text on screen, "method-select-wrapper" would parallel the "select" control, but "option" is just a third name for the same thing. I lean towards "method-select".

JohnThomson commented:
I have fixed the background color and added warning comments about FF56 not working right. I can't find a low-cost way to get rid of the pesky focus outline.
On the class names, I went with method-select-wrapper for the outer div, method-shared for the properties shared by the select itself and its options, a new method-root for the select itself, and stuck with method-option for the classes that primarily apply to one option within the select (though one of them at any given time is also applied to the select itself). Willing to consider others...I didn't like the idea of method-select for the root thing on the select, which would have ended up with a rule for select.method-select, nor for classes mainly aimed at individual options within the select.

---

Review status: 0 of 4 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1957#-:-KywGWDDOSZHgxVoDPvG:bxaidyk)*
<!-- Sent from Reviewable.io -->
